plugins/repo/Makefile

```diff
@@ -0,0 +1,26 @@
+.PHONY: build-in-docker build clean
```

Comment:
Can we include this from a common.mk in the common plugin? I'd imagine this will get repetitive.